editoast/src/tests/train_schedule.json

```
@@ -23,5 +23,7 @@
   "path_id": 40,
   "rolling_stock_id": 9,
   "timetable_id": 19,
-  "scheduled_points":  []
+  "rollingstock_version": 0,
+  "infra_version": "1",
```

This should be handled in fixtures like the `timetable_id`, `path_id`, and `rolling_stock_id`. 
Note: We could set the values of all these fields to `-1` (they must be change)

```suggestion
  "rollingstock_version": -1,
  "infra_version": "-1",
```